docs/MigrationGuide.md

```diff
@@ -12,6 +12,7 @@ To see the biggest differences please consult the [changelog](Changelog.md).
   From now import scenario with nodes from other categories than scenario category will be not allowed.
 * [#3436](https://github.com/TouK/nussknacker/pull/3436) Division by zero will cause validation error. Tests that rely on `1/0` to generate exceptions should have it changed to code like `1/{0, 1}[0]`
 * [#3473](https://github.com/TouK/nussknacker/pull/3473) JsonRequestResponseSinkFactory provides also 'raw editor', to turn on 'raw editor' add `SinkRawEditorParamName -> "true"`
+* [#3608](https://github.com/TouK/nussknacker/pull/3608) Use `ZonedDateTime` for `date-time` JsonSchema format, timezone is required now 
```

Comment:
`timezone is required now` - I think, it always was.. 
this code:
```scala
LocalDateTime.ofInstant(ZonedDateTime.parse(dateTime, DateTimeFormatter.ISO_DATE_TIME).toInstant, ZoneId.systemDefault())
```
throws exception if dateTime is without timezone

Reply:
:+1: true...